ocs_ci/ocs/constants.py

```diff
@@ -1015,3 +1015,12 @@
 
 # aws tags
 AWS_CLOUDFORMATION_TAG = 'aws:cloudformation:stack-name'
+
+# MCG namespace constants
+MCG_NS_AWS_ENDPOINT = 'https://s3.amazonaws.com'
+MCG_NS_RESOURCE = 'ns_resource'
+MCG_NS_BUCKET = 'ns-bucket'
+MCG_NS_AWS_CONNECTION = 'aws_connection'
+MCG_NS_REGION = 'us-east-2'
+MCG_NS_RESULT_DIR = '/result'
```

Comment:
MCG_NS_RESULT_DIR and MCG_NS_ORIGINAL_DIR look like test specific variables. It shouldn't be in global constants file. It can be a global variable in tests/manage/mcg/test_namespace.py.